openedx/adg/lms/applications/tests/test_admin/test_inlines.py

```diff
@@ -61,3 +70,132 @@ def test_responsibilities_work_experience_inline(work_experience):
     actual_responsibilities = WorkExperienceInline.responsibilities('self', work_experience)
 
     assert expected_responsibilities == actual_responsibilities
+
+
+@pytest.mark.django_db
+@mock.patch('openedx.adg.lms.applications.admin.modulestore')
+def test_empty_multilingual_course_group(mock_module_store):
+    """
+    Test formset is valid when no courses are added.
+    """
+    mock_module_store.get_course.return_value = mock.Mock()
+    data = {
+        'multilingual_courses-TOTAL_FORMS': '2',
+        'multilingual_courses-INITIAL_FORMS': '0',
+        'multilingual_courses-0-id': '',
+        'multilingual_courses-0-multilingual_course_group': '',
+        'multilingual_courses-0-course': '',
+        'multilingual_courses-1-id': '',
+        'multilingual_courses-1-multilingual_course_group': '',
+        'multilingual_courses-1-course': '',
+        'name': 'Test',
+        'is_prerequisite': 'on',
+    }
```

Comment:
We are repeating this at 4 places, Let's please create a fixture to return this, even if there are some changes in values, we can handle the using parameters in the fixture.

Reply:
Once these are moved in a fixture, we can easily manage all the tests with `@pytest.mark.parametrize`